test/libyul/yulSyntaxTests/tstore_warning_only_once.yul

```diff
@@ -0,0 +1,10 @@
+{
+    tstore(0, 0)
+    tstore(0, 1)
+    let x := tload(0)
+    tstore(1, 0)
+}
+// ====
+// EVMVersion: >=cancun
+// ----
```

Comment:
Oh, actually, maybe this could also use a `.sol` test with multiple contracts to make sure this is emitted once per compilation, not once per contract.